config/cable.yml

```diff
@@ -0,0 +1,14 @@
+development:
+  adapter: redis
+  url: <%= ENV.fetch("OFN_REDIS_URL", "redis://localhost:6379/1") %>
+  channel_prefix: your_application_development
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("OFN_REDIS_URL", "redis://localhost:6379/1") %>
+  channel_prefix: your_application_production
+
+test:
+  adapter: redis
+  url: <%= ENV.fetch("OFN_REDIS_URL", "redis://localhost:6379/1") %>
+  channel_prefix: your_application_test
```

Comment:
If you look into config/environments then you see that we actually use "redis://localhost:6380/0" in staging and production.

Reply:
Thanks for the heads up! I changed the URL's for production and staging.